Dockerfile

```diff
@@ -1,34 +1,46 @@
-# if you are on Mac M1 please add --platform=linux/amd64 after FROM below
 FROM node:14-alpine AS base
 RUN mkdir -p /usr/src/app && \
     addgroup -g 1001 appuser && \
     adduser -S -u 1001 -G appuser appuser && \
     chown -R appuser:appuser /usr/src/app && \
     chmod -R +x  /usr/src/app && \
     apk update && \
-    apk upgrade && \
-    apk add --no-cache bash git curl
-
+    apk upgrade
 
 FROM base AS dependencies
 WORKDIR /usr/src/app
 USER 1001
 COPY package.json package-lock.json ./
 RUN npm i
-COPY tsconfig.json babel.config.js webpack.config.js .eslintrc.js ./
-COPY docker/apply/forms-json ./docker/apply/forms-json
-COPY --chown=appuser:appuser ./src ./src/
-
+COPY package-lock.json package-lock-cache.json
```

Comment:
Since `npm i` would now be the last step of the stage, it does not cache effectively. Docker caching works by detecting if the next step has any changes to the checksum, if not, it means the current step can use the cache